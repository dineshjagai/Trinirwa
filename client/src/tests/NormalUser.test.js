import React from 'react';
import renderer from 'react-test-renderer';
import NormalUser from '../components/NormalUser';

test('Test NomralUser snapshot', () => {
  const component = renderer.create(<NormalUser />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
