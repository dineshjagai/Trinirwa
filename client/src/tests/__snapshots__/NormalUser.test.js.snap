// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test NomralUser snapshot 1`] = `
<div>
  <h1>
    Normal User
  </h1>
</div>
`;
